### PR TITLE
[HOCS-4838] Run on Node 16.x

### PR DESCRIPTION
**Pull request by robertdeniszczyc2**
This change allows the enables the module to build and run on Node version 16.x (current LTS).

Also included:
- package-lock.json uses lockfileVersion 2
- Fixed ESLint warnings, and marked several false positives as TODO
- README updated

### CODE REVIEW

**Comment by nickmcmahon01** [comment]
🥇

**Comment by robertdeniszczyc2** [comment]
The diff for the package-lock.json file is so big because it's now using lockfile version 2 rather than 1, and the structure of the lockfile has changed.

**Review by LiamMacP — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:14-alpine as base
+FROM node:16.16.0-alpine3.15 as base
```
Comment (nickmcmahon01) [inline Dockerfile:1]: can this be node:16-alpine please

**Review by nickmcmahon01 — DISMISSED** [review]

File: Dockerfile (hunk 1 above)
Comment (nickmcmahon01) [inline Dockerfile:1]:
```suggestion
FROM node:16-alpine as base
```

**Pull request merged**